template-share-extension/package.json

```diff
@@ -40,7 +40,7 @@
     "@lumx/react": "^2.0.1",
     "axios": "0.21.1",
     "lodash": "4.17.21",
-    "lumapps-sdk-js": "^1.0.1",
+    "lumapps-sdk-js": "^1.0.0",
```

Comment:
This version is bugged, please use the previous one (0.0.44 as far as i remember)